### DISCUSSION TITLE
Compare current best price and historical lowest price against the regional pricing rather the US price.

### DISCUSSION THREAD

**Discussion by theodorax99** [IsThereAnyDeal/AugmentedSteam#1935] [Ideas]
Hi there! 

It seems like the Current best and Historical low is compared against the US price in Steam rather than my actual region. 

For example here. Even though the current price is 9.23$ in my region. It shows that the current best is 14.89$ in a different store as it's comparing against the U.S price which is 19.79$ as I take this screenshot.

![chrome_bvHHuGb2vW](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/173655819/744c32c4-4461-4d3c-bff7-280ea7cba76a)

Here, you can see the current price is lower, yet Current Best and Historical low is again showing US prices. 

![chrome_j88Ly14OH2](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/173655819/925bbb82-0eca-409a-91a7-a60566e7fa3f)

It would really helpful if there's an option or something where you can adjust this in the settings as currently, the Current Best and Historical low is pretty much useless for anyone in regions where prices are usually cheaper compared to the US.

Same for Epic games as it also have regional pricing now.

**Comment by candela97** [comment]
For currencies not supported by ITAD, prices are converted from USD, see https://github.com/IsThereAnyDeal/AugmentedSteam/issues/482.
